src/components/Buttons/ButtonGroup.jsx

```diff
@@ -14,6 +14,10 @@ const ButtonGroup = ({ size, value, isCollapsed = false, onChange, className, ch
             {Children.map(children, (child, index) => {
                 const buttonProps = { size };
 
+                if (child.props.isHidden) {
+                    return;
```

Comment:
@khomyakov Are you sure React is not complaining because the child is `undefined`? Can we return `null`?